libs/perun/services/src/lib/html-escape.service.ts

```diff
@@ -0,0 +1,124 @@
+import { Injectable } from '@angular/core';
+import { addHook, sanitize } from 'dompurify';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class HtmlEscapeService {
+  escapeDangerousHtml(html: string): string {
+    const config = {
+      ALLOWED_TAGS: [
+        'a',
+        'article',
+        'aside',
+        'b',
+        'blockquote',
+        'br',
+        'button',
+        'caption',
+        'center',
+        'cite',
+        'decorator',
+        'del',
+        'details',
+        'div',
+        'em',
+        'footer',
+        'h1',
+        'h2',
+        'h3',
+        'h4',
+        'h5',
+        'h6',
+        'header',
+        'i',
+        'img',
+        'kbd',
+        'label',
```

Comment:
I just noticed that if we have allowed tags `ul/ol`, then we should also allow `li` tag - without this tag list items cannot be added to the un/ordered list.
```suggestion
        'label',
        'li'
```